app/models/package.js

```diff
@@ -48,6 +48,10 @@ export default DS.Model.extend({
     return this.get('packageType.name');
   }),
 
+  changedNotes: Ember.computed('notes', function(){
```

Comment:
I noticed there was an Ember.computed.alias method, would that be appropriate to this case ?

Reply:
Yes, I agree with you @patrixr, It will work according to me.